project/grpc.sbt

```diff
@@ -1,2 +1,2 @@
-addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "0.4.2")
-addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.4") // ALPN agent
\ No newline at end of file
+addSbtPlugin("com.lightbend.akka.grpc" % "sbt-akka-grpc" % "0.7.1")
```

Comment:
Do we want to remove the grpc library entirely?

Reply:
I had created [this issue](https://github.com/cfpb/hmda-platform/issues/3019) for removing it completely...just to make sure the codebase isn't using it . At the moment I just wanted to fix this because it is a blocker for https://github.com/cfpb/hmda-platform/issues/2974

Reply:
`rate-limit` and `census-api` do currently actually include `grpc` endpoints. I will add that to the ticket.